cerbero/packages/package.py

```diff
@@ -111,6 +111,8 @@ class PackageBase(object):
     resources_postinstall = 'postinstall'
     resources_postremove = 'postremove'
     conflicts = ''
+    replaces = ''
```

Comment:
Since those are debian specific, prepend debian_ to the variable name. Is there an equivalent for RPM ?